homeassistant/components/microsoft_face/__init__.py

```diff
@@ -313,10 +341,12 @@ async def call_api(self, method, function, data=None, binary=False, params=None)
             )
             raise HomeAssistantError(answer["error"]["message"])
 
-        except aiohttp.ClientError:
-            _LOGGER.warning("Can't connect to microsoft face api")
+        except aiohttp.ClientError as err:
+            _LOGGER.error("Can't connect to microsoft face api with error: %s", err)
+            raise
```

Comment:
The aiohttp errors won't be caught in `async_setup`. Probably remove the raise here, since we're raising HomeAssistantError below.

Reply:
I don't advise to remove this "raise" to give feedback to the user when invoking the service through the frontend.

Reply:
We're already raising `HomeAssistantError` below. Isn't that enough?

Reply:
Why wasn't the existing error enough?

Reply:
No, because that error is lacking error information given by the API.

I just wanted to pitch in with a small improvement, but this level of scrutiny is not very encouraging.

Reply:
Ok. Then we need to catch these errors in `async_setup`.